src/main/resources/xml/schema/2020-1/nordic-html5.rng

```diff
@@ -4376,138 +4376,52 @@
     </define>
 
     <define name="classes">
-        <ref name="classes.without.images" />
+        <a:documentation>Allow all classes, except certain classes that have a special meaning.</a:documentation>
+        <a:documentation>Classes with a special meaning will be handled explicitly where relevant.</a:documentation>
         <zeroOrMore>
-            <choice>
-                <value>image</value>
-                <value>image-series</value>
-            </choice>
-        </zeroOrMore>
-        <ref name="classes.without.images" />
-        <zeroOrMore>
-            <choice>
-                <value>image</value>
-                <value>image-series</value>
-            </choice>
-        </zeroOrMore>
-        <ref name="classes.without.images" />
-    </define>
-
-    <define name="classes.without.images">
-        <zeroOrMore>
-            <a:documentation> Common classes in use </a:documentation>
-            <choice>
-                <!-- Common classes in use -->
-                <value>address</value>
-                <value>annoref</value>
-                <value>annotation</value>
-                <value>answer</value>
-                <value>answer_1</value>
-                <value>asciimath</value>
-                <value>backcover</value>
-                <value>bdo</value>
-                <value>box</value>
-                <value>bridgehead</value>
-                <value>byline</value>
-                <value>cite</value>
-                <value>colophon</value>
-                <value>cover</value>
-                <value>dateline</value>
-                <value>definition</value>
-                <value>docauthor</value>
-                <value>emptyline</value>
-                <value>exercisenumber</value>
-                <value>exercisepart</value>
-                <value>fig-desc</value>
```

Comment:
`fig-desc` should still be in the list I think. I haven't looked through all the other classes, but I noticed it when looking at #432.